plugins/conf_maps/lib/api/KalturaConfMaps.php

```diff
@@ -105,17 +110,41 @@ public function validateForInsert($propertiesToSkip = array())
 		parent::validateForInsert($propertiesToSkip);
 	}
 
-	public function validateContent()
+	public function validateAndHandleContent()
 	{
-		$contentArray = json_decode($this->content, true);
-		if(!$contentArray)
+		$content = json_decode($this->content, true);
+		if (is_null($content))
```

Comment:
Since in some cases json_decode returns false, better check jusr if($content)

Reply:
use if (json_last_error() != JSON_ERROR_NONE)